bin/dev

```diff
@@ -0,0 +1,2 @@
+require('dotenv/config')
+require('./../server/index.js')
```

Comment:
What do you use this file for?

Reply:
At one point I was trying to use environmental variables, then went a different direction and forgot to delete the related files.